src/onboarding/steps.ts

```diff
@@ -40,6 +41,8 @@ export interface OnboardingProps {
   supportedBiometryType: BIOMETRY_TYPE | null
   skipVerification: boolean
   numberAlreadyVerifiedCentrally: boolean
+  showRecoveryPhrase: boolean
+  showCloudBackupFakeDoor: boolean
```

Comment:
It doesn't appear that showCloudBackupFakeDoor is used in steps. Is it necessary here?

Reply:
It's not used in `_getStepInfo`, but it _is_ used as a parameter throughout onboarding. If we remove it from here, we'd need an extra call to `getOnboardingExperimentParams` in `ProtectWallet`. On the one hand, it's an extra (potentially unnecessary call) but on the other, it means we wouldn't be "polluting" the `OnboardingProps` interface.

Reply:
I think OnboardingProps should just be for props that determine the steps, not props that dictate internal workings of individual steps.

Reply:
Sure, I think I agree -- happy to move it over.